recipes/ffmpeg/all/conanfile.py

```diff
@@ -125,6 +126,7 @@ class FFMpegConan(ConanFile):
         "with_freetype": True,
         "with_openjpeg": True,
         "with_openh264": True,
+        "with_openvino": True,
```

Comment:
I don't think it's a good idea to enable openvino in ffmpeg by default, there are already too many dependencies enabled in ffmpeg recipe, and it will leak to other recipes like opencv.